.github/workflows/check.yml

```diff
@@ -0,0 +1,25 @@
+name: Check for updates
+
+on:
+  schedule:
+    - cron: '52 3/6 * * *'
+
+jobs:
+  check:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: docker://ghcr.io/flathub/flatpak-external-data-checker:latest
+        env:
+          GIT_AUTHOR_NAME: Endless External Data Checker
+          GIT_COMMITTER_NAME: Endless External Data Checker
+          GIT_AUTHOR_EMAIL: os@endlessos.org
+          GIT_COMMITTER_EMAIL: os@endlessos.org
+          # TODO: set GITHUB_TOKEN. Cannot use secrets.GITHUB_TOKEN, fails with:
+          # github.GithubException.GithubException: 403 {'message': 'Resource not accessible by integration', 'documentation_url': 'https://docs.github.com/rest/reference/repos#create-a-fork'}
```

Comment:
I don't quite understand this because I've successfully used the `GITHUB_TOKEN` to open a PR in another action, namely https://github.com/flathub/org.vim.Vim/blob/master/.github/workflows/auto-update.yaml.

Reply:
I believe from some googling that the token is supposed to be able to create PRs if and only if the action comes from the repo itself, not a fork. But this is not a fork…

Reply:
Oh, actually the failure is because fedc is trying to **create** a fork because it does not believe that it has push access to the origin repo. Weird.